src/renderers/dom/fiber/wrappers/ReactDOMFiberInput.js

```diff
@@ -97,6 +98,39 @@ var ReactDOMInput = {
         getCurrentFiberStackAddendum,
       );
 
+      if (props.type === 'checkbox') {
+        warning(
+          false,
+          '%s contains an input of type %s with a value prop, but not a checked prop. ' +
+            'If you use value instead of checked with a controlled checkbox ' +
+            "it looks like its working but it's not. The input remains " +
```

Comment:
`it looks like its working but it's not. The input remains `

Change to "it looks like **_it's_** working but it's not."